### PR TITLE
docs: update SSO docs with latest CLI commands

### PR DESCRIPTION
**Pull request by hf**
Updates the SSO docs to use the CLI commands.

### CODE REVIEW

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (hf) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:22]: Change the version here!

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (J0) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:22]:
```suggestion
This guide requires the use of the [Supabase CLI](/docs/guides/cli). Please make sure you're using version v1.46.4 or higher. You can use `supabase -v` to see the currently installed version.
```
Probably should update this once dogfooding is done, guessing that it might be v1.46.4 after any patches

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (J0) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:202]: /nit 
```suggestion
With exception of the unique user ID, SAML does not require any other attributes in the assertion. Identity providers can be configured so that only select user information is shared with your project.
```

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (J0) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:226]: On v1.46.1 the flag is labelled as  `--attribute-mapping-file` instead of `--attribute-mapping` do double check before merging

Comment (J0) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:226]: CleanShot 2023-03-30 at 20 12 18@2x for quick reference

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (J0) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:342]: /nit consider rephrasing

```suggestion
Unfortunately, it is difficult to use SAML when developing on your machine. This is a security limitation by identity providers which do not allow setting the SAML Assertion Consumer Service URL to non-HTTPS URLs, or they completely disallow the use of `localhost` in such settings.
```

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (kangmingtay) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:226]: should be `--attribute-mapping-file` based on the latest version

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (kangmingtay) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:344]: would they be able to use `https://samltest.id/` for this if they want to try out the entire flow though?

**Review by kangmingtay — CHANGES_REQUESTED** [review]
left mostly comments around re-phrasing things better but there are 2 things we should check:
1. Command inputs / outputs from this doc reflects the commands in the CLI
2. Support for IDP initiated flow

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (hf) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:281]: Let's leave it to see if people actually run into the problem and are not able to figure it out on their own. I doubt it's a common occurrence...

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (hf) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:344]: Not really, it's somewhat doable but it's a bit too complex to go into. We can maybe have a separate blog post about it -- testing SAML locally or something. I wouldn't go into it in this doc as it's very little value.

Comment (hf) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:344]: This is why I wanted to create the saml tool, but haven't had enough time to finish it.

Comment (kangmingtay) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:344]: should we just remove this section then? not too sure if it provides much value if there aren't clear instructions on how to go about doing so

**Review by kangmingtay — APPROVED** [review]

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx
Comment (kangmingtay) [reply apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:178]: supported but seems quite tricky to set up - will probably need to branch off to a separate guide here in the future

**Pull request merged**
